packages/babel-parser/test/fixtures/experimental/record-and-tuple/invalid-type-bar-record/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": [["recordAndTuple", { "syntaxType": "invalid" }]],
+  "throws": "Unexpected token (1:1)"
```

Comment:
Here we should throw an error similar to `@babel/generator`'s for the invalid option.

Reply:
Good call, how about this? (used the pipelineOperator option checks for a reference here).

```
'recordAndTuple` requires `syntaxType` option whose value should be one of: 'hash', 'bar'
```

Reply:
Yes :+1:

Reply:
done :)